cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -225,22 +85,22 @@ spec:
 #    osd:
   storage: # cluster level storage configuration and selection
     useAllNodes: true
-    useAllDevices: false
+    useAllDevices: true
     deviceFilter:
     location:
     config:
       # The default and recommended storeType is dynamically set to bluestore for devices and filestore for directories.
       # Set the storeType explicitly only if it is required not to use the default.
       # storeType: bluestore
-      databaseSizeMB: "1024" # this value can be removed for environments with normal sized disks (100 GB or larger)
-      journalSizeMB: "1024"  # this value can be removed for environments with normal sized disks (20 GB or larger)
-      osdsPerDevice: "1" # this value can be overridden at the node or device level
+      # databaseSizeMB: "1024" # this value can be removed for environments with normal sized disks (100 GB or larger)
```

Comment:
How about changing these comments to be "uncomment if the disks are less than 100 GB"

Reply:
agreed